app/controllers/pageflow/application_controller.rb

```diff
@@ -21,8 +21,14 @@ class ApplicationController < ActionController::Base
 
     rescue_from ActiveRecord::RecordNotFound do
       respond_to do |format|
-        format.html { render :file => Rails.root.join('public/404.html'), :status => :not_found }
-        format.any(:json, :css) { head :not_found }
+        format.html do
+          begin
+            render file: Rails.public_path.join('pageflow', 'error_pages', '404.html'), status: :not_found
```

Comment:
I'm still unsure if we should have a test for this? The line itself is covered. We could add an expection to the `EntriesController#show` spec that some text from the generated file shows up. For the rescue clause below, I see no way to get test coverage. A test could temporarily rename the `404.html` file in the dummy app. But I'm not sure it's worth it.